packages/backend/src/utils/npm.ts

```diff
@@ -120,6 +109,50 @@ class Command {
     }
   }
 
+  private async shouldBeUpdated(packageJson: any): Promise<boolean> {
+    const queryUrl = this.getSingleGenQueryURL(packageJson.name);
+    const npmJsModules = await npmFetch.json(queryUrl);
+    const npmJsModule: any = _.get(npmJsModules, "objects.[0]");
+    return npmJsModule ? npmJsModule.package.version !== packageJson.version : false;
+  }
+
+  public getGlobalNodeModulesPath(): string {
+    return this.globalNodeModulesPath;
+  }
+
+  public async getPackagesMetadata(query = "", author = "") {
```

Comment:
```suggestion
  public async getPackagesMetadata(query = "", author = ""): Promise<any> {
```

Reply:
it is the same in this case

Reply:
I know, but suitable eslint rule seems like turned off and the other apis have returns type in function signature